### PR TITLE
Update Algorithm Where Art Thou

### PR DESCRIPTION
**Pull request by alayek**
Refer #1094

### CODE REVIEW

File: Algorithm-Where-Art-Thou.md [hunk 1]
```diff
@@ -60,7 +69,7 @@ whereAreYou([{ first: "Romeo", last: "Montague" }, { first: "Mercutio", last: nu
 
 :rocket: [Run Code](https://repl.it/CLmh/0)
 
-### Second
```
Comment (atjonathan) [inline Algorithm-Where-Art-Thou.md]: Add a Code Explanation

File: Algorithm-Where-Art-Thou.md [hunk 2]
```diff
@@ -85,7 +94,7 @@ whereAreYou([{ first: "Romeo", last: "Montague" }, { first: "Mercutio", last: nu
 
 :rocket: [Run Code](https://repl.it/CLmi/0)
 
-### Third
```
Comment (atjonathan) [inline Algorithm-Where-Art-Thou.md]: Add a Code Explanation

**Comment by Rafase282** [comment]
Please add the code explanations and links for the solutions.

**Comment by alayek** [comment]
@Rafase282 I am not getting what you mean by 
> links for the solutions

**Comment by Rafase282** [comment]
@alayek each section should have:

```
#### Relevant Links

- [Link Title 1](http://example.com)
- [Link Title 2](http://example.com)
- [Link Title 3](http://example.com)
```

**Pull request closed without merging**